js/views/playingScreen.js

```diff
@@ -229,6 +246,12 @@ var PlayingScreen = React.createClass({
           playerState={this.props.playerState}
           isLiveStream={this.props.isLiveStream} />
       </div>
+      {showUnmute ? <button className="oo-playing-screen oo-unmute"
```

Comment:
Same comment about attributes like the one above

Reply:
Done.